.github/workflows/release.yml

```diff
@@ -0,0 +1,47 @@
+name: Production Build
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+jobs:
+  release:
+    env:
+      LATEST_GO: 1.19.x # version used for release
+    strategy:
+      matrix:
+        go: [ 1.19.x ]
+        os: [ ubuntu-latest ]
+        arch:
+        - arm64
+        - amd64
+    name: ${{ matrix.os }}/go${{ matrix.go }}
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup go
+        uses: actions/setup-go@v3
+        with:
+          go-version: ${{ matrix.go }}
+
+      - run: make test
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Set env
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+
+      - run: make push PLUGIN_NAME=ghcr.io/${{ github.repository_owner }}/${{ github.repository }} PLUGIN_TAG=$RELEASE_VERSION-${{ matrix.arch }} ARCH=${{ matrix.arch }}
```

Comment:
now we're changing the plugin's registry to github and changing the versioning as well, so we should also update the README too?

Reply:
done